radis/spectrum/operations.py

```diff
@@ -1046,7 +1046,7 @@ def offset(s, offset, unit, name=None, inplace=False):
     return s
 
 
-def get_baseline(s, var="radiance", Iunit=None):
+def get_baseline(s, var="radiance", wunit="nm", Iunit=None):
```

Comment:
btw, here I think the default for wunit should be `"default"`, and if `"default"` overwrite with the Spectrum default waveunit 
```
s.get_waveunit()
```